doitlive.py

```diff
@@ -22,9 +22,12 @@
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
+from termutils import raw_mode
 
 import click
-from click import echo, style, secho, getchar
+click.disable_unicode_literals_warning = True  # this fixes #12
```

Comment:
I'm not sure this is the proper fix; from the docs, it appears that click wants users to explicitly use the `u` prefix.